Documentation/Signoffs/MasterMCU.md

```diff
@@ -0,0 +1,41 @@
+MASTER MICROCONTROLLER
+------------------------
+### Function of the Subsystem
+The master microcontroller (MMCU) must receive data from four seperate local microcontrollers that interface with the
+sensors in other subsystems. This MMCU will read serial data from each local MCU and then send that data to the PC to
+be processed in the simulation.
+
+### Specifications & Constraints
```

Comment:
You need to consider the code that you intend to run. You don't have to have it written but you must identify a worst case and show that this system will still function in that context.